resources/scponly_user.rb

```diff
@@ -0,0 +1,99 @@
+#true To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+
+resource_name :scponly_user
+
+default_action :create
+
+property :write_dir, String, default: 'incoming'
+property :public_key, String
+property :private_key, String
+property :chroot, [true, false], default: true
+
+action :create do
+  run_context.include_recipe 'osl-scponly::default'
+
+  if new_resource.chroot
+    user new_resource.name do
+      gid 'scponly'
+      manage_home true
+      shell '/usr/sbin/scponlyc'
+    end
+
+    directory "/home/#{new_resource.name}//#{new_resource.write_dir}"do 
+      owner 'root'
+      group 'scponly'
+    end
+
+    altroot = "/home/#{new_resource.name}"
+    binaries = %w(/bin/chgrp /bin/chmod /bin/chown /bin/ln /bin/ls /bin/mkdir /bin/mv /bin/rm /bin/rmdir /usr/bin/scp /usr/sbin/scponlyc)
```

Comment:
Should binaries be a property? it seems like since this is a scponly recipe it should only have the binaries necessary for that. I may trim down this list of binaries once it's working, this is just a list I found in their documentation.

Reply:
I still think this should be a property with a sane default.